test/performance/requirements.txt

```diff
@@ -0,0 +1,4 @@
+kubernetes==30.1.0
+PyYAML==5.4.1
+PyYAML==6.0.2
+urllib3==1.26.5
```

Comment:
FYI, installing just kubernetes 30.1.0 brings in a lot of other stuff. Here is what I found:

```shell
(test5) mspreitz@mjs13 test5 % pip list
Package            Version
------------------ -----------
cachetools         5.5.0
certifi            2024.8.30
charset-normalizer 3.3.2
google-auth        2.34.0
idna               3.8
kubernetes         30.1.0
oauthlib           3.2.2
pip                24.2
pyasn1             0.6.0
pyasn1_modules     0.4.0
python-dateutil    2.9.0.post0
PyYAML             6.0.2
requests           2.32.3
requests-oauthlib  2.0.0
rsa                4.9
six                1.16.0
urllib3            2.2.2
websocket-client   1.8.0
```

Are you sure you want to downgrade urllib3?

Reply:
Fixed the issue with `PyYAML` package - it was a typo.  The command `pip list` shows all the package installed in your system. Are you sure that all those packages didn't already exist before you installed the `Kubernetes` module?  I see a long list of package on my testing machine even without installing the `Kubernetes` module first.

Reply:
Yes, I see only `pip` itself before installing kubernetes.

```
mspreitz@mjs13 ~ % python3 -m venv test5
mspreitz@mjs13 ~ % . test5/bin/activate
(test5) mspreitz@mjs13 ~ % pip list
Package Version
------- -------
pip     24.2
(test5) mspreitz@mjs13 ~ % pip install kubernetes==30.1.0
Collecting kubernetes==30.1.0
  Using cached kubernetes-30.1.0-py2.py3-none-any.whl.metadata (1.5 kB)
Collecting certifi>=14.05.14 (from kubernetes==30.1.0)
  Using cached certifi-2024.8.30-py3-none-any.whl.metadata (2.2 kB)
Collecting six>=1.9.0 (from kubernetes==30.1.0)
  Using cached six-1.16.0-py2.py3-none-any.whl.metadata (1.8 kB)
Collecting python-dateutil>=2.5.3 (from kubernetes==30.1.0)
  Using cached python_dateutil-2.9.0.post0-py2.py3-none-any.whl.metadata (8.4 kB)
Collecting pyyaml>=5.4.1 (from kubernetes==30.1.0)
  Using cached PyYAML-6.0.2-cp312-cp312-macosx_11_0_arm64.whl.metadata (2.1 kB)
Collecting google-auth>=1.0.1 (from kubernetes==30.1.0)
  Using cached google_auth-2.34.0-py2.py3-none-any.whl.metadata (4.7 kB)
Collecting websocket-client!=0.40.0,!=0.41.*,!=0.42.*,>=0.32.0 (from kubernetes==30.1.0)
  Using cached websocket_client-1.8.0-py3-none-any.whl.metadata (8.0 kB)
Collecting requests (from kubernetes==30.1.0)
  Using cached requests-2.32.3-py3-none-any.whl.metadata (4.6 kB)
Collecting requests-oauthlib (from kubernetes==30.1.0)
  Using cached requests_oauthlib-2.0.0-py2.py3-none-any.whl.metadata (11 kB)
Collecting oauthlib>=3.2.2 (from kubernetes==30.1.0)
  Using cached oauthlib-3.2.2-py3-none-any.whl.metadata (7.5 kB)
Collecting urllib3>=1.24.2 (from kubernetes==30.1.0)
  Using cached urllib3-2.2.2-py3-none-any.whl.metadata (6.4 kB)
Collecting cachetools<6.0,>=2.0.0 (from google-auth>=1.0.1->kubernetes==30.1.0)
  Using cached cachetools-5.5.0-py3-none-any.whl.metadata (5.3 kB)
Collecting pyasn1-modules>=0.2.1 (from google-auth>=1.0.1->kubernetes==30.1.0)
  Using cached pyasn1_modules-0.4.0-py3-none-any.whl.metadata (3.4 kB)
Collecting rsa<5,>=3.1.4 (from google-auth>=1.0.1->kubernetes==30.1.0)
  Using cached rsa-4.9-py3-none-any.whl.metadata (4.2 kB)
Collecting charset-normalizer<4,>=2 (from requests->kubernetes==30.1.0)
  Using cached charset_normalizer-3.3.2-cp312-cp312-macosx_11_0_arm64.whl.metadata (33 kB)
Collecting idna<4,>=2.5 (from requests->kubernetes==30.1.0)
  Using cached idna-3.8-py3-none-any.whl.metadata (9.9 kB)
Collecting pyasn1<0.7.0,>=0.4.6 (from pyasn1-modules>=0.2.1->google-auth>=1.0.1->kubernetes==30.1.0)
  Using cached pyasn1-0.6.0-py2.py3-none-any.whl.metadata (8.3 kB)
Using cached kubernetes-30.1.0-py2.py3-none-any.whl (1.7 MB)
Using cached certifi-2024.8.30-py3-none-any.whl (167 kB)
Using cached google_auth-2.34.0-py2.py3-none-any.whl (200 kB)
Using cached oauthlib-3.2.2-py3-none-any.whl (151 kB)
Using cached python_dateutil-2.9.0.post0-py2.py3-none-any.whl (229 kB)
Using cached PyYAML-6.0.2-cp312-cp312-macosx_11_0_arm64.whl (173 kB)
Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Using cached urllib3-2.2.2-py3-none-any.whl (121 kB)
Using cached websocket_client-1.8.0-py3-none-any.whl (58 kB)
Using cached requests-2.32.3-py3-none-any.whl (64 kB)
Using cached requests_oauthlib-2.0.0-py2.py3-none-any.whl (24 kB)
Using cached cachetools-5.5.0-py3-none-any.whl (9.5 kB)
Using cached charset_normalizer-3.3.2-cp312-cp312-macosx_11_0_arm64.whl (119 kB)
Using cached idna-3.8-py3-none-any.whl (66 kB)
Using cached pyasn1_modules-0.4.0-py3-none-any.whl (181 kB)
Using cached rsa-4.9-py3-none-any.whl (34 kB)
Using cached pyasn1-0.6.0-py2.py3-none-any.whl (85 kB)
Installing collected packages: websocket-client, urllib3, six, pyyaml, pyasn1, oauthlib, idna, charset-normalizer, certifi, cachetools, rsa, requests, python-dateutil, pyasn1-modules, requests-oauthlib, google-auth, kubernetes
Successfully installed cachetools-5.5.0 certifi-2024.8.30 charset-normalizer-3.3.2 google-auth-2.34.0 idna-3.8 kubernetes-30.1.0 oauthlib-3.2.2 pyasn1-0.6.0 pyasn1-modules-0.4.0 python-dateutil-2.9.0.post0 pyyaml-6.0.2 requests-2.32.3 requests-oauthlib-2.0.0 rsa-4.9 six-1.16.0 urllib3-2.2.2 websocket-client-1.8.0
(test5) mspreitz@mjs13 ~ % 
(test5) mspreitz@mjs13 ~ % pip list
Package            Version
------------------ -----------
cachetools         5.5.0
certifi            2024.8.30
charset-normalizer 3.3.2
google-auth        2.34.0
idna               3.8
kubernetes         30.1.0
oauthlib           3.2.2
pip                24.2
pyasn1             0.6.0
pyasn1_modules     0.4.0
python-dateutil    2.9.0.post0
PyYAML             6.0.2
requests           2.32.3
requests-oauthlib  2.0.0
rsa                4.9
six                1.16.0
urllib3            2.2.2
websocket-client   1.8.0
(test5) mspreitz@mjs13 ~ % 
```

Reply:
ok, I see! I updated the `requirements.txt` file to only include the Kubernetes module